### PR TITLE
Update to winreg 1.2.0

### PR DESCRIPTION
**Pull request by BobBuehler**
Updated docs to match docs in the source.

### CODE REVIEW

**Comment by RX14** [comment]
I'll check this PR out tomorrow, or later tonight if I have time.

File: winreg/winreg.d.ts [hunk 1]
```diff
@@ -1,44 +1,65 @@
...
     /**
-     * Create a new Winreg instance with the given options.
-     * @param options options object
+     * Creates a registry object, which provides access to a single registry key.
+     * Note: This class is returned by a call to ```require('winreg')```.
+     *
+     * @public
+     * @class
+     *
+     * @param {object} options - the options
```
Comment (RX14) [inline winreg/winreg.d.ts]: This documentation is a duplicate of the docs on the `Winreg.Options` class.

**Comment by RX14** [comment]
Apart from that minor docs duplication, this looks good to me.

**Comment by BobBuehler** [comment]
Thanks for the speedy review. I have removed the duplication while maintaining some detail that wasn't present in both locations.

**Comment by RX14** [comment]
:+1: 

looks great to me, hope there won't be any more incompatible updates.

**Pull request merged**